### ISSUE TITLE
Remove institutional logos from Projects page

### ISSUE THREAD

**Issue by KZzizzle** [ITISFoundation/osparc-issues#195]
User story: As a user on sparc.science, I would like to remove the logos on the projects pages. 
##### Please leave your contact information
**Name**
Charles Horn

**Email**
chorn@pitt.edu

**Can you provide a short description of what you were doing?**
I am concerned that we do not have permission for use of institutional logos

**Can you provide specific and detailed description?**
See Projects pages.

**How would you like the experience to improve?**
remove the logos for now until this issue is discussed by CCB

**Click the box if you would like us to keep you in the loop as we work to resolve this issue.**
Keep me updated
##### Tell us where you found the issue and what you were trying to do.
**My feedback is about:**
General Feedback

**Comment by KZzizzle** [comment]
➤ DRC SIMCORE commented:

Charles Horn how urgent/critical is this? Ideally the contributors would remove the logos themselves which would take time to communicate. In theory anyone with access to contentful could do it, but I'm not sure if that steps on anyone's toes..

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

I see this as urgent; I would recommend not making this live until we flush out the process more completely.

**Comment by KZzizzle** [comment]
➤ DRC SIMCORE commented:

Sonia Sharma could I assign this to you? Jeremy Lecoeur is there an automated way to remove all logos so Sonia doesn't have to remove logos one at a time?

**Comment by KZzizzle** [comment]
➤ David Nickerson commented:

not documentation, so probably not something Sonia Sharma can easily do. Not sure Sonia Sharma is aware of the non-documentation (help) content being pulled from Contentful and how that is being managed.

**Comment by KZzizzle** [comment]
➤ DRC SIMCORE commented:

followers consulted with Leonardo Guercio and Abby Paulson : we will wait for weigh-in from the CCB before we do anything about the logos.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

CCB Review: 
Nina: not sure that there is something in the contracts that NIH could publicize stuff. This may already be addressed in the fine print in contracts. 
Andrew: I will double check with Kristina; she brought this up.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

In the fair use and international rights act, this may be ok. 

Assiging back to Andrew to validate.

**Comment by KZzizzle** [comment]
➤ Andrew Weitz commented:

Leonardo Guercio
From Kristi:
"The DRC should look into where each logo was obtained from (e.g. university website) and check what the usage rights are.  Permission to reuse the logos should be requested if necessary.  If the DRC chooses not to ask permission, then they need to be prepared to remove one immediately if a cease/desist is received.  If there is no guidance AND it's trademarked/copyrighted, then the default should be fair use doesn't apply."
Kristi is more worried about the private sector and nonprofit logos than the university ones.

**Comment by KZzizzle** [comment]
➤ Joost Wagenaar commented:

Would hate to see them go as I think they add a lot to the site and the insight in the scale of sparc. But easiest is to remove if legally, there seems to be a concern.

**Comment by KZzizzle** [comment]
➤ Ellen M. Goldstein - CX commented:

Abby Paulson - we also think that they add a lot visually to the site. Who is responsible for determining this?

**Comment by KZzizzle** [comment]
➤ Nina Preuss commented:

Recommend multi tier: 1. Keep University logos since likely OK, and be prepared to remove if we receive cease/desist request.

**Comment by KZzizzle** [comment]
➤ Nina Preuss commented:

2. Low risk: Apinatomy (Bernard), BF Disocver, BF Data Management Platform, TissueMapper, ScaffoldFitter, InterLex, ScaffoldMaker, TissueMaker, Vesselucida, NeuroLucida, SciCrunch are in house; OpenCor, Scigraph are open source)  .  3.  Higher Risk: Physiomproject and Protocols.io, should be removed and receive an email.  Once approved, uploaded again.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

Abby Paulson  will reach out to the identified High risk projects and confirm they are ok with the logo use. If not, will move forward to remove.

**Comment by KZzizzle** [comment]
➤ Andrew Weitz commented:

Medium risk (not universites, but all SPARC awardees): Feinstein Institute for Medical Research, California Medical Innovations Institute, Cleveland Clinic, g-tec Medical, Mass General Hospital, Micro-Leads Medical

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

Ah, we went initially through the Resources page, but you're correct. The Projects page (as this was initially written) also has a lot... hm. Nina Preuss there are more - https://sparc.science/data?type=sparcAward

**Comment by KZzizzle** [comment]
➤ Nina Preuss commented:

Andrew Weitz since they are all SPARC Awardees, and thus have OTs and thus Administrative POCs, if Abby drafts an email, can Kristina send it to all the POCs, or authorize Charles to provide the POC  list to Abby to email on behalf of DRC?  It will be much more efficient then slogging through websites to hit or miss to find the right POCs.

**Comment by KZzizzle** [comment]
➤ Andrew Weitz commented:

Nina Preuss, let me check with Kristi.  I believe she is OOO until Friday, so it could take a couple days to get an answer.  My hunch is that the institutional official POCs we have are not the right contacts.  Our POCs are in the Contracts/Grants offices of universities, whereas I believe the logos are managed by universities' Trademark/Licensing offices.

**Comment by KZzizzle** [comment]
➤ Nina Preuss commented:

Agreed but they could more efficiently route the request

**Comment by KZzizzle** [comment]
➤ Andrew Weitz commented:

Nina Preuss, I followed up with Kristi.  She thinks the DRC should be the one to follow up on this, since they own the site.  However, she brought up a good point about the institutional officials being overwhelmed now due to covid-19, so probably not a great time to contact them about logos.  She suggested emailing the PIs and asking them to forward the request to their institutional public affairs & communications offices.

**Comment by KZzizzle** [comment]
➤ Nina Preuss commented:

We do have that list. Charles Horn if Abby Paulson drafts an email about logo permissions, can your send it to the PIs to forward? Or, feel free to draft it yourself and run it by Kristina

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

Yes please draft it and I'll send it. I'll be traveling tomorrow (driving) but I could get to this over the weekend ... but per Andrew's point above, I believe Universities are overwelmed at the moment and will be unlikely to respond quickly.

**Comment by KZzizzle** [comment]
➤ Nina Preuss commented:

Abby Paulson can you draft an email?

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

I'll draft something and send it along for review.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

Draft option >

Hello, 

I am reaching out with you in regards to the NIH-funded SPARC Program. I work with the Data and Resource Center which manages the sparc.science site. On the site, we would like to use the logos of instutions and partners who have generated data and resources that are now available. If you object to us using your logo for this purpose, please reply and we will make the appropriate adjusments. Thank you and have a great day. 

Best, 
[name]

Charles HornNina PreussAndrew Weitz what do you think of the above? Short and to the point is the approach; but we could be more long-winded if you think necessary...

**Comment by KZzizzle** [comment]
➤ Nina Preuss commented:

I would just say we will remove your logo or modify it as you instruct. Otherwise it looks good to me. I believe you offered to send it? please copy us for traceability.

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

Abby Paulson for the above message, who will be receiving this? The SPARC award PI?

**Comment by KZzizzle** [comment]
➤ Nina Preuss commented:

if you have an Administrative contact that's better than the PI. If you don't, I would suggest you modify the message and ask them to please forward as appropriate.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

Kristi had suggested, 
suggested emailing the PIs and asking them to forward the request to their institutional public affairs & communications offices.
according to Andrew in the above comment. 

Based on that I have made the following adjustment: 

I am reaching out to you in regards to the NIH-funded SPARC Program. I work with the Data and Resource Center which manages the sparc.science site. On the site, we would like to use the logos of instutions and partners who have generated data and resources that are now available. Please forward this request to your institutional public affairs and communication offices, as appropriate. If they object to us using the logo for this purpose, please reply and we will remove or modify it as you instruct. Thank you and have a great day.

**Comment by KZzizzle** [comment]
➤ Andrew Weitz commented:

^ That sounds good to me

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

This seems fine to me. Would suggest that I include Andrew and Kristi on the CC to make this more official. Shall I move forward? (sending to PIs and MPIs)

**Comment by KZzizzle** [comment]
➤ Kristina Faulk commented:

Hi Charles Horn , Andrew Weitz and Abby Paulson ! I think it would be better to use the generic sparc mailbox as the "cc" because the respective PIs may not know Andrew or me if they haven't had to work with us directly/frequently. Please use NIH-CF_SPARC@mail.nih.gov if we are in agreement.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

That makes sense to me.

**Comment by KZzizzle** [comment]
➤ Nina Preuss commented:

Good point

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

I have reworded this.
Does this work?
-----------------------------------------------------
CC: NIH-CF_SPARC@mail.nih.gov

Dear SPARC Project PI:

I hope you are doing well. 

I am reaching out to you because I have been working with the SPARC Data and Resource Center to develop the sparc.science portal (https://sparc.science/).

Importantly, on the site, we are trying to showcase each project team. As part of this we are showing institutional logos (https://sparc.science/data?type=sparcAward). 

We are wondering if there is any objection to showing these logos on the site. Could you please forward this request to your institutional public affairs and communication offices, as appropriate?

If they object to using the logo for this purpose, please reply and we will remove or modify the logo display.

Thank you.

Charles
Member of the Change Control Board for the SPARC Portal

--------
Charles Horn, Ph.D.
Associate Professor
University of Pittsburgh
Hillman Cancer Center - Research Pavilion, 1.19d
5117 Centre Avenue
Pittsburgh, PA 15213  USA

**Comment by KZzizzle** [comment]
➤ Andrew Weitz commented:

Looks good to me, Charles Horn!  One small change I'd make is to rename "sparc.science portal" to "SPARC Portal"

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

Andrew Weitz ... Thanks. I'll send it soon. ... One issue that should be considered as a future step is how to show the multi-institutional nature of SPARC projects since a logo does not capture this (this seems to be one of our core values that differentiates us from a list of traditional projects, ie, R01s)

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

It has been sent to the 55 PIs and MPI contacts!

**Comment by KZzizzle** [comment]
➤ Andrew Weitz commented:

Thanks Charles Horn!  And great point about capturing the multi-institutional aspects of some projects.  Maybe submit a separate Wrike ticket for that?

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

Thanks. Andrew Weitz, this is now submitted as a separate ticket.

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

I did the following:
1. sent email to all PIs last week to request permission
2. received some responses (all positive), and cataloged these in a private Airtable and Box.com folder (shared with both Andrew Weitz and Kristina Faulk
3. three PIs requested that we change their logos to current ones (I sent these to Leonardo Guercio to do this)
4. a few requests to update the project summary (which I’ll do directly) because the NIH Reporter is out of date; I’m also making these updates in the consortium Airtable.

**Comment by KZzizzle** [comment]
➤ Kristina Faulk commented:

Hi Charles Horn ! Please hold on project summary update as the info on the Portal and RePORTer should match. The PIs must initiate the RePORTer updates. Please provide their names so NIH may contact them with instructions on how to do this. Once that process is complete then the Portal version can be udpated as well.

**Comment by KZzizzle** [comment]
➤ Kristina Faulk commented:

As always, your help with this is invaluable! 😀

**Comment by KZzizzle** [comment]
➤ Nina Preuss commented:

Fantastic!

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

Hi Kristina Faulk ... I can do this; this one was the Vitality Napadow project. Keep in mind that Matt Ward's project does not have an abstract on Reporter and awhile ago Matt supplied his abstract to me to update the Airtable (which was subsequentely mined for data that was pushed to contentful). Should I take his abstract down also? In addition, Janet Keast indicated that her abstract on Reporter is not correct (and I have not changed in Contentful). These are the three PIs with Reporter issues.

**Comment by KZzizzle** [comment]
➤ Kristina Faulk commented:

HI Charles Horn! Please leave the abstract up. I'm looking into this on the NIH side because all three have provided info that failed to make it into RePORTer. I'm not sure why or how this process failed, but hope to resolve it soon.

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

Hi Kristina Faulk ... ok. no problem.
